doc/sphinx-guides/source/user/find-use-data.rst

```diff
@@ -96,19 +95,21 @@ Ingested files can be downloaded in several different ways.
 
 - A subset of the columns of the data
 
+
 .. _rsync_download:
 
-Downloading a Dataverse Package via rsync
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Downloading a Dataverse Package via rsync + SSH
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Uploads are rsync+ssh; downloads are rsync w\out ssh (aka - published package files do not need to be encrypted in transit or use authentication).